docs/admin/deployment/index.md

```diff
@@ -46,10 +46,9 @@ Deploy NeMo Curator on Kubernetes clusters using Dask Operator, GPU Operator, an
 :::{grid-item-card} {octicon}`server;1.5em;sd-mr-1` Slurm Deployment
 :link: admin-deployment-slurm
 :link-type: ref
-Run NeMo Curator on Slurm clusters with shared filesystems. Covers job scripts, Dask cluster setup, module execution, monitoring, and advanced Python-based job submission.
+Run NeMo Curator on Slurm clusters with shared filesystems using Singularity/Apptainer containers. Covers job scripts, Dask cluster setup, module execution, monitoring, and advanced Python-based job submission.
```

Comment:
I remember a few months ago someone outside Nvidia was having trouble running Curator using Singularity. Maybe we should omit unless we know it works.